src/rdata/mod.rs

```diff
@@ -132,4 +132,9 @@ rdata_types! {
             Zonemd<O>,
         }
     }
+    naptr::{
```

Comment:
@partim: Should this be ordered alphabetically (with rfc1035 as an exception first) ?

Reply:
I'm going to re-order this because for me it's like staring at a wonky painting on the wall ;-)